hnginternship5/ziki
# Did Inline css for blog-details page and 404 page

#### What does this PR do?
* It inline for Blog-details and created 404 page
#### Description of Task to be completed?
* Blog-details page `blog-detail.html`
* Styling is added to inline
* 404 page `404.html` in templates folder
* Styling is added to inline
#### How should this be manually tested?
* Open the site on your browser
* Type localhost/blog-details 
* Type localhost/404
* You should see any of the page

#### Screenshots (if appropriate)

*******404 page
***************************
These are my Browser Screenshot you can compare with design
![Screen Shot 2019-04-21 at 12 40 44 AM](https://user-images.githubusercontent.com/38491485/56463562-d33e5080-63ce-11e9-8b01-7d9bbcd176c5.png)


*************Blog-details-page
************************************
![Screen Shot 2019-04-21 at 12 40 04 AM](https://user-images.githubusercontent.com/38491485/56463570-fe28a480-63ce-11e9-8b5e-0d7a8e8925c9.png)


#### Questions/Comments:
